CHANGELOG.md

```diff
@@ -1,935 +1,17 @@
-# [6.8.8](https://github.com/dcodeIO/protobuf.js/releases/tag/6.8.8)
+# Changelog
 
-## Fixed
-[:hash:](https://github.com/dcodeIO/protobuf.js/commit/3001425b0d896d14188307cd0cc84ce195ad9e04) Persist recent index.d.ts changes in JSDoc<br />
+## [6.9.0](https://www.github.com/protobufjs/protobuf.js/compare/6.8.8...v6.9.0) (2020-03-11)
 
-# [6.8.7](https://github.com/dcodeIO/protobuf.js/releases/tag/6.8.7)
 
-## Fixed
-[:hash:](https://github.com/dcodeIO/protobuf.js/commit/e8449c4bf1269a2cc423708db6f0b47a383d33f0) Fix package browser field descriptor ([#1046](https://github.com/dcodeIO/protobuf.js/issues/1046))<br />
-[:hash:](https://github.com/dcodeIO/protobuf.js/commit/996b3fa0c598ecc73302bfc39208c44830f07b1a) Fix static codegen issues with uglifyjs3<br />
-[:hash:](https://github.com/dcodeIO/protobuf.js/commit/a06317139b92fdd8c6b3b188fb7b9704dc8ccbf1) Fix lint issues / pbts on windows<br />
```

Comment:
It didn't do a good job of detecting the starting point of the old CHANGELOG, we will be submitting a PR to fix this.